grails-app/views/manageLayers/field.gsp

```diff
@@ -108,11 +109,13 @@
                                             edit</g:link>
                                     </td>
                                     <td>
-                                        <a onclick="return confirmDelete('${item.id}');"
-                                           class="btn btn-sm btn-default">
-                                            <i class="glyphicon glyphicon-remove"></i>
-                                            delete
-                                        </a>
+                                        <g:if test="${item.enabled}">
+                                            <a onclick="return confirmDelete('${item.id}','${item.name}' );">
+                                                <i class="glyphicon glyphicon-remove"></i> </a>
+                                        </g:if>
+                                        <g:else>
+                                            <i class="glyphicon glyphicon-ban-circle" data-toggle="tooltip" data-placement="top" title="You are not allowed to delete a disabled field."></i>
```

Comment:
Remove this new logic. There is no reason to make admin users go through extra steps to delete a field.

Reply:
I have read this deletion code. It won't do anything if the field is disabled. I don't want to change this logic, so I disable the delete button

Reply:
Create an new issue or fix it please. Deleting a disabled item will not break anything.

Reply:
Fixed